src/components/security-code-error/security-code-error-controller.ts

```diff
@@ -18,6 +18,13 @@ export function securityCodeInvalidGet(req: Request, res: Response): void {
       Date.now() + getCodeEnteredWrongBlockDurationInMinutes() * 60000
     ).toUTCString();
   }
+
+  if (req.query.actionType === SecurityCodeErrorType.EmailMaxRetries) {
```

Comment:
This is going to put on a lock when a user enters more than 5 incorrect codes during registration. Should this SecurityCodeErrorType be `ChangeSecurityCodesEmailMaxRetries` instead?